### DISCUSSION TITLE
Spring 核心技术

### DISCUSSION THREAD

**Discussion by giscus** [joeljhou/joeljhou.github.io#13] [Announcements]
# Spring 核心技术

Spring 核心技术 核心技术 Spring 框架的核心技术主要包括： 依赖注入（dependency injection），也称为控制反转（IoC） 事件（events） 资源（resources） 国际化（i18n） 数据验证（validation） 数据绑定（data binding） 类型转换（type conversion） SpEL（Sp...

https://www.geekyspace.cn/spring-framework/core/

**Comment by joeljhou** [comment]
❤️
